playground/src/Playground.tsx

```diff
@@ -1,13 +1,27 @@
-import React, { ReactNode } from 'react';
+import React, { CSSProperties } from 'react';
 import { LiveProvider, LiveEditor, LiveError, LivePreview } from 'react-live';
 import dracula from 'prism-react-renderer/themes/dracula';
 import toJSXString from 'react-element-to-jsx-string';
 
 const styles = {
   preview: {
     padding: '1rem',
-    border: '1px solid #e1e4e8',
-  },
+    border: '1px solid #E1E4E8',
```

Comment:
It's a bit too much I think on dark mode, I would go for something like `hsla(0, 0%, 50%, .2)` , normally it's working on dark/light, WDYT ?